resolwe/flow/tests/test_filtering.py

```diff
@@ -400,6 +407,20 @@ def test_filter_text(self):
         self._check_filter({'text': 'rst'}, [])
         self._check_filter({'text': 'process'}, self.data)
 
+        # Order by newest first by default.
+        self._check_filter({'text': 'Hello'},
+                           # "hello", "hellodata", "Hello Hello", "hello hello"
+                           [self.data[3], self.data[2], self.data[1], self.data[0]],
+                           check_ordering=True)
+
+        # Order by relevance.
+        # Ordering by empty string should override DataViewSet's default ordering (-created)
+        # and return as ordered by Elastic Search.
+        self._check_filter({'text': 'Hello', 'ordering': ''},
+                           # "Hello Hello", "hello hello", "hello", "hellodata"
+                           [self.data[1], self.data[0], self.data[3], self.data[2]],
+                           check_ordering=True)
```

Comment:
Does this look robust enough to overcome randomness?

Reply:
Nope

Reply:
Rename data objects to a single word, eg. Five and Six